gcloud-java-compute/src/main/java/com/google/gcloud/compute/OperationId.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.gcloud.compute;
+
+/**
+ * Interface for Google Compute Engine operation identities.
+ */
+public interface OperationId {
+
+  /**
+   * Returns the name of the project.
+   */
+  String project();
+
+  /**
+   * Returns the name of the operation resource.
+   */
+  String operation();
+
+  /**
+   * Returns a fully qualified URL to the entity.
```

Comment:
Nit: I think it would be slightly clearer to say "operation" rather than "entity"